### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by saifullah767**
# Validation
Form email will not accept Upper case
Email Must be in Lower case

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -224,3 +224,16 @@ function popupProject(y) {
...
+
+const cform = document.querySelector('.contact-form');
+const errMessage = document.querySelector('.validate');
+const mail = document.querySelector('.contactme');
+
+cform.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (mail.value !== mail.value.toLowerCase()) {
+    errMessage.style.display = 'block';
+  } else {
```
Comment (Donard97) [inline index.js:236]: - [ ] You did an amazing job adding the error message but kindly remove it near the submit button as it was part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md). Keep shining 💪 

> - If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

>

File: project_one.html [hunk 2]
```diff
@@ -222,11 +223,13 @@ <h2>Contact me</h2>
             />
             <input
               type="email"
+              class="contactme"
               name="user_email"
               id="email"
               placeholder="Enter your email"
               required
             />
+            <div class="validate" > Email must in Lower case </div>
```
Comment (Donard97) [inline project_one.html]: - [OPTIONAL] I suggest you to wrap the text in a p tag following semantic HTML and kindly fix the misspelling. Keep coding 💪

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @saifullah767 ,  👋 

You did a tremendous job so far! 👏 💯 

There are some issues that you still need to work on to go to the next project but you are almost there! 💪 

hgiphy

## Highlights ✅ 

✔️ The Readme file is professional
✔️  PR is descriptive
✔️ Code is clean
✔️ Validation works perfectly

## Required Changes ♻️

- [ ] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by phelian23 — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**